imp_developer.py

```diff
@@ -272,56 +326,414 @@ def __base64_encode(str):
+            data = None
+            if not error:
+                for device in response.get('data'):
+                    devgrp = device["relationships"].get("devicegroup")
+                    if (devicegroup_id == None
+                        or (devgrp and devgrp["id"] == devicegroup_id)):
+                        devices.append(device)
+                next_link = response["links"].get("next")
+                if next_link == link:
+                    link = None
+                else:
+                    link = next_link
+            else:
+                return response, error
+
+        return devices, error
+
+    @staticmethod
+    def get_devicegroup(token, devicegroup_id):
+        url = PL_IMPCENTRAL_API_URL_V5 + "devicegroups/" + devicegroup_id
+        response, code = HTTP.get(token, url=url)
+        error = ImpCentral.handle_http_response(response, code)
+        return response.get("data"), error
+
+    @staticmethod
+    def get_deployment(token, deployment_id):
+        url = PL_IMPCENTRAL_API_URL_V5 + "deployments/" + deployment_id
+        response, code = HTTP.get(token, url=url)
+        error = ImpCentral.handle_http_response(response, code)
+        return response.get("data"), error
+
+    @staticmethod
+    def get_device(token, device_id):
+        response, code = HTTP.get(token,
+            PL_IMPCENTRAL_API_URL_V5 + "devices/" + device_id)
+        error = ImpCentral.handle_http_response(response, error)
+        return response, error
+
+
+    @staticmethod
+    def create_product(token, product_name):
+        url = PL_IMPCENTRAL_API_URL_V5 + "products"
+        data = json.dumps({
+            "data": {
+                "type": "product",
+                "attributes": {
+                    "name": product_name,
+                    "description": "Product created from sublime plugin"
+                }
+            }
+        })
+        response, code = HTTP.post(token, url, data, headers=HttpHeaders.DEFAULT)
+
+        error = ImpCentral.handle_http_response(response, code)
+
+        return response.get("data"), error
+
+    @staticmethod
+    def create_devicegroup(token, product_id, devicegroup_name):
+        url = PL_IMPCENTRAL_API_URL_V5 + "devicegroups"
+        data = json.dumps({
+            "data": {
+                "type": "development_devicegroup",
+                "attributes": {
+                    "name": devicegroup_name,
+                    "description": "Devicegroup created from sublime plugin"
+                },
+                "relationships": {
+                    "product": {
+                        "type": "product",
+                        "id": product_id
+                    }
+                }
+            }})
+
+        response, code = HTTP.post(token, url, data)
+        error = ImpCentral.handle_http_response(response, code)
+        return response.get("data"), error
+
+    @staticmethod
+    def create_logstream(token):
```

Comment:
`create_logstream ` -> `create_log_stream` (here and everywhere else please!)

Reply:
fixed everywhere